framework/pom.xml

```diff
@@ -7,12 +7,27 @@
   <version>1.0.0-SNAPSHOT</version>
   <properties>
     <testng.version>6.4</testng.version>
-    <log4j.version>1.2.17</log4j.version>
+    <slf4j.version>1.7.26</slf4j.version>
+    <logback.version>1.2.9</logback.version>
     <jackson.version>2.10.0</jackson.version>
-    <drill.version>${env.DRILL_VERSION}</drill.version>
-    <hadoop.version>${env.HADOOP_VERSION}</hadoop.version>
-    <runtimeDepenencies>${env.DRILL_HOME}/conf</runtimeDepenencies>
+    <guava.version>30.1.1-jre</guava.version>
+    <drill.version>2.0.0-SNAPSHOT</drill.version>
```

Comment:
Is it better that this stops being an env var?

Reply:
It is bound to the Drill version (or the hadoop version used in Drill). There is no reason for it to be an environment variable.

Reply:
So if I want to run the test framework against a Drill 1.20.1 instead of a Drill 2.0.0-SNAPSHOT then I should edit the pom file?

Reply:
Correct. The idea of the test framework is to test the latest version, so the pom.xml have to be updated accordingly, if the version is changed. We assume that previous versions were already tested so no need for more flexibility here. And in case of different Drill branches (i.e. hadoop 2) we can create corresponding branches of Drill Test Framework.